Northstar.Client/mod/scripts/vscripts/ui/_menus.nut

```diff
@@ -178,6 +178,23 @@ void function UICodeCallback_ToggleInGameMenu()
 	}
 }
 
+
+void function TimeoutHandler()
+{
+	//Singleplayer "Continue" counts as finished loading and shouldn't be affected
+	EndSignal( uiGlobal.signalDummy, "LevelShutdown" ) // Cancels thread if map is unloaded
+
+	float maxLoadTime = GetConVarFloat( "ns_max_load_time" )
+	wait maxLoadTime
+	if ( uiGlobal.isLoading == true ) //Disconnects client after 30 seconds of loading
+	{
+		printt( format( "TimeoutHandler disconnected because loading took too long (%i seconds)", maxLoadTime ) )
```

Comment:
Since maxLoadTime is a float, does this work?
Might have to change it to this:
```suggestion
		printt( format( "TimeoutHandler disconnected because loading took too long (%f seconds)", maxLoadTime ) )
```